force-app/main/default/permissionsets/PMDM_Deliver.permissionset-meta.xml

```diff
@@ -190,6 +265,33 @@
         <object>ServiceDelivery__c</object>
         <viewAllRecords>false</viewAllRecords>
     </objectPermissions>
+    <objectPermissions>
+        <allowCreate>true</allowCreate>
+        <allowDelete>false</allowDelete>
```

Comment:
And everything on Participant.

Reply:
Deliver permset does not have delete permissions on any object. @meggraySFDO , Do we need to give users with Deliver perm set access to delete participant records